openwisp_controller/config/static/config/css/admin.css

```diff
@@ -230,6 +231,11 @@ div.change-form #device_form > div > div.inline-group {
 .change-form #device_form div.inline-group.tab-content > .tabular {
   margin-top: -1em;
 }
+#devicelocation-0-2-heading {
+  font-size: 16px;
+  text-transform: uppercase;
+  padding: 15px;
```

Comment:
Isn't the problem generalized? I was wondering if we should look for a generic solution for all the headers. Are you sure we need to change only this one?

Reply:
Myabe, we can generalize this using the following rule 

```css
.inline-related fieldset.module h4 {
  font-size: 16px;
  text-transform: uppercase;
  padding: 15px;
}
```

But, this will make heading of all fieldsets bigger.

Reply:
I don't see a side-effect of this now in openwisp-monitoring.